newt/newt.go

```diff
@@ -128,7 +126,7 @@ func newtCmd() *cobra.Command {
 	newtCmd.PersistentFlags().BoolVarP(&newtHelp, "help", "h",
 		false, "Help for newt commands")
 	newtCmd.PersistentFlags().BoolVarP(&newtEscapeShellCmds, "escape", "",
-		false, "Apply Windows escapes to shell commands")
+		runtime.GOOS == "windows", "Apply Windows escapes to shell commands")
```

Comment:
Does it really matter to turn it on by default for Windows here, if above you have `util.EscapeShellCmds = newtEscapeShellCmds`?

Reply:
I wanted to keep default behavior same as in 1.6